components/script/dom/servoparser/async_html.rs

```diff
@@ -633,18 +635,20 @@ impl TreeSink for Sink {
         node
     }
 
-    fn has_parent_node(&self, node: &Self::Handle) -> bool {
-        self.sender.send(ToTokenizerMsg::HasParentNode(node.id)).unwrap();
-        match self.receiver.recv().expect("Unexpected channel panic in html parser thread.") {
-            ToSinkMsg::HasParentNode(result) => result,
-            _ => unreachable!(),
-        }
+    fn has_parent_node(&self, _: &Self::Handle) -> bool {
+        unreachable!();
     }
 
-    fn associate_with_form(&mut self, target: &Self::Handle, form: &Self::Handle) {
+    fn associate_with_form(&mut self,
+        target: &Self::Handle,
+        form: &Self::Handle,
+        nodes: (&Self::Handle, Option<&Self::Handle>)) {
```

Comment:
Nit: block indenting, the arguments list closing paren and opening brace should be on their own line.